projects/angular-showcase/src/app/examples/tooltip-showcase/tooltip.md

```diff
@@ -50,3 +50,46 @@ The overlay is displayed:
 
 - on top the icon question.
 - on bottom the icon question.
+
+<br>
+
+### Trigger type
+
+```html
+<sbb-tooltip [triggerType]="'click'">
```

Comment:
Please rename triggerType to trigger.
Also shorten this example to `trigger="click"`, which is equivalent to `[trigger]="'click'"`